src/Uno.UI/UI/Xaml/DebugSettings.cs

```diff
@@ -5,6 +5,8 @@ namespace Microsoft.UI.Xaml
 {
 	public sealed partial class DebugSettings
 	{
+		public LayoutCycleTracingLevel LayoutCycleTracingLevel { get; set; }
```

Comment:
Let's document this one, in xmldoc, as well as in the performance doc.

Reply:
I'm not sure how it relates to perf. Can you elaborate please?

Reply:
Right, it's not really performance. Let's place it there then: https://platform.uno/docs/articles/common-issues-all-ides.html, and explain the error message, what happens when the property is enabled, what to do with the results.